test/mock.utils.ts

```diff
@@ -24,21 +24,36 @@ export const loadMockSession = (mockLoadSessionFunction: jest.Mock<typeof loadSe
 };
 
 export const loadCompanyAuthenticatedSession = (mockLoadSessionFunction: jest.Mock<typeof loadSession>,
-                                                companyNumber?: string): void => {
+                                                companyNumber?, email?): void => {
```

Comment:
should this be `companyNumber?: string, email?: string`

Reply:
We need these to be null for some tests.

Reply:
think he meant missing :string types

Reply:
I know - typing was creating issues here - but I think I have found a way around this.